app/reducers/keychainReducer.js

```diff
@@ -31,7 +31,7 @@ const keychainReducer = (state = initialState, action) => {
       )
     case SET_KEYCHAIN:
       return state.withMutations((st) => {
-        for (let key of action.keychain) {
+        for (const key of action.keychain) {
```

Comment:
As long as `key` isn't modified, it works fine. It's in the spec and also specifically noted as an example [on MDN](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/for...of). I did do a little digging and saw it's not well implemented in Firefox, but not relevant here.

I agree at first sight it looks absurd, but it does fix a linting issue and it's also kind of cool in a way.. ;)